mp4parse/src/lib.rs

```diff
@@ -565,6 +571,32 @@ fn read_moov<T: Read>(f: &mut BMFFBox<T>, context: &mut MediaContext) -> Result<
     Ok(())
 }
 
+fn read_mvex<T: Read>(src: &mut BMFFBox<T>) -> Result<MovieExtendsBox> {
+    let mut iter = src.box_iter();
+    let mut duration: Option<u64> = None;
```

Comment:
The compiler should be able to infer this type from the signature of `read_mehd` so you only need this if you especially want to document it locally. Up to you.